backend/src/Designer/Services/Implementation/ImageUrlValidationService.cs

```diff
@@ -0,0 +1,27 @@
+using System.Threading.Tasks;
+using Altinn.Studio.Designer.Enums;
+using Altinn.Studio.Designer.Services.Interfaces;
+
+namespace Altinn.Studio.Designer.Services.Implementation;
+
+public class ImageUrlValidationService : IImageUrlValidationService
+{
+    private readonly IImageUrlPolicyValidator _imageUrlPolicyValidator;
+    private readonly ImageClient _imageClient;
+
+    public ImageUrlValidationService(IImageUrlPolicyValidator imageUrlPolicyValidator, ImageClient imageClient)
+    {
+        _imageUrlPolicyValidator = imageUrlPolicyValidator;
+        _imageClient = imageClient;
+    }
+
+    public async Task<ImageUrlValidationResult> ValidateUrlFullyAsync(string url)
+    {
+        if (!_imageUrlPolicyValidator.IsAllowed(url))
+        {
+            return ImageUrlValidationResult.NotValidImage;
+        }
+
+        return await _imageClient.ValidateUrlAsync(url);
+    }
```

Comment:
I did not quite understand the `Fully` part of `ValidateUrlFullyAsync` at first, especially when called from a different file.

What do you think about changing the original `ValidateUrlAsync` to `ValidateResponseAsync` - and `ValidateUrlFullyAsync` to `ValidateUrlAsync`? 
```suggestion
    public async Task<ImageUrlValidationResult> ValidateUrlAsync(string url)
    {
        if (!_imageUrlPolicyValidator.IsAllowed(url))
        {
            return ImageUrlValidationResult.NotValidImage;
        }

        return await _imageClient.ValidateResponseAsync(url);
    }
```

Reply:
Fixed, I named it: ValidateRequestResponseAsync.